cypress.config.js

```diff
@@ -0,0 +1,14 @@
+import { defineConfig } from 'cypress';
+
+export default defineConfig({
+  e2e: {
+    env: {
+      baseUrl: 'https://annaawojtowicz.github.io/social-media-client/',
```

Comment:
```suggestion
      baseUrl: '/',
```

Please use Cypress' built in web server, since this will test the "hot" code in the branch, not the "cold" code that is deployed to hosting. The idea with testing is that we test before we deploy, not after - so that we know whether our changes have broken something.

Reply:
Fixed